server/chart_config.json

```diff
@@ -28,9 +28,9 @@
     "statsVars": ["Count_Person_Employed"],
     "isOverview": true,
     "relatedChart": {
-      "title": "Percentage of People Employed",
+      "title": "Employment Rate",
```

Comment:
Hi @beets , @shifucun -- I do not know if I made the right call here. Although "Employed" is a subclass of "InLaborForce", this stat is not very helpful as it is simply 1 - unemploymentRate, which we already have.

% of People Employed, on the other hand, is interesting to those who think the way "unemployment" is measured (% of labor force) is unfair.